core/src/main/java/org/frankframework/align/Json2Xml.java

```diff
@@ -117,6 +121,15 @@ private static Set<String> getNamesOfXsdChildElements(XSComplexTypeDefinition co
 				.collect(Collectors.toSet());
 	}
 
+	private static List<XSParticle> getXsdChildParticles(XSComplexTypeDefinition complexTypeDefinition) {
+		XSTerm term = complexTypeDefinition.getParticle().getTerm();
+		if (!(term instanceof XSModelGroup modelGroup)) {
```

Comment:
Ik zou 'm omgedraaid beter te lezen vinden (term instanceof en dan in de braces iets doen met wat je zojuist gecast hebt)

Reply:
Hier allicht beter inderdaad. Copy-paste van een method erboven waarin meer gebeurde en een "early return" meer gerechtvaardigd was.